shared/templates/create_audit_rules_usergroup_modification.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_usergroup_modification.py
+#        generate template-based checks for user/group modification rules
+
+
+from template_common import FilesGenerator, UnknownTargetError
+
+import os
+import re
+
+class AuditRulesUserGroupModificationGenerator(FilesGenerator):
+    def generate(self, target, args):
+        path = args[0]
+        name = re.sub('[-\./]', '_', os.path.basename(path))
+        if target == "oval":
+            self.file_from_template(
+                "./template_OVAL_audit_rules_usergroup_modification",
+                {
+                    "%NAME%":	name,
+                    "%PATH%":	re.sub("/", "\/", path)
```

Comment:
Bizarrely enough this works. I am really baffled as to why. Is it because of `Unlike Standard C, all unrecognized escape sequences are left in the string unchanged, i.e., the backslash is left in the string. (This behavior is useful when debugging: if an escape sequence is mistyped, the resulting output is more easily recognized as broken.)` in https://docs.python.org/2.0/ref/strings.html ?